src/main/java/com/epam/machine/controller/ClientsOrdersServlet.java

```diff
@@ -1,18 +1,30 @@
 package com.epam.machine.controller;
 
+import com.epam.machine.entity.Offer;
+import com.epam.machine.service.OfferServiceImpl;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
 
 public class ClientsOrdersServlet extends HttpServlet {
+
+    OfferServiceImpl offerService = new OfferServiceImpl();
+
     @Override
-    public void doGet (HttpServletRequest request, HttpServletResponse response) {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) {
         try {
-            request.setAttribute("fail", 0);
+            HttpSession session = request.getSession();
+            String login = session.getAttribute("login").toString();
+            List<Offer> offers = offerService.get(login);
```

Comment:
Plz, could u rename method `get` to `getAllByEmail`? And why u still use login instead of email?

Reply:
Admin doesn't have email, only login

Reply:
Admin? Class calls `ClientsOrdersServlet` so it's probably working with client?